src/rust/engine/rule_graph/src/lib.rs

```diff
@@ -441,7 +441,7 @@ impl<'t, R: Rule> GraphMaker<'t, R> {
           params: params.clone(),
           reason: if params.is_empty() {
             format!(
-              "No rule was available to compute {}. Maybe declare it as a RootRule({})?",
+              "No rule was available to compute {}. Maybe declare RootRule({})?",
```

Comment:
I like using the Markdown convention of surrounding text that's meant to be code in ``, although I'm sure we don't do this consistently in other error messages.